scripts/schedulers/pbs.py

```diff
@@ -20,7 +20,10 @@ def get_metadata(self, db_running_task: SchedulerTask, task_logger: Logger):
         cmd = f"qstat -f -F json -x {db_running_task.job_id}"
 
         out, err = self._run_command_and_wait(cmd, shell=True)
-        json_dict = json.loads(out)
+        # special treatment after nurion added GPU replated variable in qstat that is a single \ which will cause json parser unable to read
```

Comment:
replated?

Removes '\\' as nurion added a GPU ? that is a single '\' which causes json parser issues. 


In general comments should be active voice with the action at the start - I still can't figure out what replated is lol.

Reply:
typo..... it was supposed to be "related" 😂